torch/csrc/jit/runtime/static/passes.cpp

```diff
@@ -950,6 +921,113 @@ void FuseListUnpack(std::shared_ptr<torch::jit::Graph>& graph) {
   }
 } // namespace jit
 
+
+// Similar to FuseListUnpack but the operators results are being accessed by get_item rather than ListUnpack
+void FuseGetItemUnpack(std::shared_ptr<torch::jit::Graph>& graph, const c10::FastMap<c10::Symbol, c10::Symbol> unfused_to_fused) {
+  // replacement contains (old_node, new_node, list_unpack_node)
```

Comment:
It would probably be clearer to define a struct rather than use a tuple.

Reply:
changed it to struct